src/frontend/src/components/public/StatusChangeDropDown/index.tsx

```diff
@@ -16,14 +16,22 @@ function StatusChangeDropDown(props: StatusChangeDropDownProps) {
     return isMPORole();
   };
 
+  const isCPO = () => {
+    return isCPORole();
+  };
+
   return (
     <>
       <div>Status</div>
       <div className="dropdownSatusContainer">
-        {isMPO() &&
-        props.mode === 'view' &&
-        statusList[props.pia.status || 'Completed'].Priviliges.MPO.changeStatus
-          .length !== 0 ? (
+        {(isMPO() &&
```

Comment:
I believe we have states do do this to avoid too many if/else.  Look into utlis/status.ts that manages what dropdown each user can change status to.